cypress/integration/parallel-4/deliveryRecords.spec.ts

```diff
@@ -16,43 +12,47 @@ import {
 	DATE_FNS_INPUT_FORMAT,
 } from '../../../shared/dates';
 import { singleContributionsAPIResponse } from '../../../client/fixtures/singleContribution';
+import {
+	guardianWeeklyPaidByCard,
+	homeDelivery,
+} from '../../../client/fixtures/productBuilder/testProducts';
 
 describe('Delivery records', () => {
 	beforeEach(() => {
 		signInAndAcceptCookies();
 
 		cy.intercept('GET', '/api/me/mma?productType=Weekly', {
 			statusCode: 200,
-			body: toMembersDataApiResponse(guardianWeeklyCurrentSubscription),
+			body: toMembersDataApiResponse(guardianWeeklyPaidByCard()),
 		}).as('product_detail');
 
-		cy.intercept('GET', '/api/delivery-records/A-S00293857', {
+		cy.intercept('GET', '/api/delivery-records/*', {
```

Comment:
Why are we not passing in the sub id now?

Reply:
the sub ID changed, and it seemed fragile to have it be hardcoded in the url when we can wildcard it away. If you think the logic of the test means we should only intercept that specific sub maybe we could specify it from the test

Reply:
good point about it being unstable...I do wonder though whether we should make it obvious that it's the correct sub id but from the looks of it we're only dealing with one sub id in the test anyway

Reply:
there is another test that has two subs, but the logic is basically around going directly to the page or not and still works